### ISSUE TITLE
confusing instance list output

### ISSUE THREAD

**Issue by fmoor** [edgedb/edgedb-cli#573]
- The note under the `edgedb instance list` table `Only portable packages shown here, use --deprecated-install-methods to show docker and package installations.` doesn't seem to be accurate. The docker instances on my machine are listed in the table.
- In the Kind column `remote` doesn't seem to be accurate since these are running in containers on my machine. 
- In the Installation column (when `--deprecated-install-methods` is used) `remote` seems misleading. These are local instances created from the new universal packages. 

Steps to Reproduce:
```bash
$ edgedb instance list
┌────────┬──────────────┬─────────────────┬────────────────────────────────────────────────┬─────────────┐
│ Kind   │ Name         │ Port            │ Version                                        │ Status      │
├────────┼──────────────┼─────────────────┼────────────────────────────────────────────────┼─────────────┤
│ local  │ edgedb_azure │ 10767           │ 1.0-rc.2+c328744                               │ not running │
│ local  │ foobar       │ 10700           │ 1.0-rc.2+c328744                               │ running     │
│ local  │ zzzzz        │ 10768           │ 1.0-rc.2+c328744                               │ not running │
│ remote │ edgedb_bench │ localhost:10765 │ 1.0-rc.1+g7bd25de57.cv                         │ up          │
│ remote │ tryit        │ localhost:10766 │ 1.0-rc.2+d2021111101.g4f8ba34d6.cv202111100000 │ up          │
└────────┴──────────────┴─────────────────┴────────────────────────────────────────────────┴─────────────┘
Only portable packages shown here, use `--deprecated-install-methods` to show docker and package installations.

$ docker ps
CONTAINER ID   IMAGE                 COMMAND                  CREATED       STATUS             PORTS                                                     NAMES
dc2cf95ee6b9   edgedb/edgedb:1-rc2   "docker-entrypoint.s…"   12 days ago   Up About an hour   5656/tcp, 0.0.0.0:10766->10766/tcp, :::10766->10766/tcp   edgedb_tryit
7c9b767ec62a   edgedb/edgedb:1-rc1   "docker-entrypoint.s…"   6 weeks ago   Up About an hour   5656/tcp, 0.0.0.0:10765->10765/tcp, :::10765->10765/tcp   edgedb_edgedb_bench

$ edgedb instance list --deprecated-install-methods
┌──────────────┬───────┬──────────┬──────────────┬─────────┐
│ Name         │ Port  │ Version  │ Installation │ Status  │
├──────────────┼───────┼──────────┼──────────────┼─────────┤
│ edgedb_bench │ 10765 │ 1-rc1    │ docker       │ running │
│ tryit        │ 10766 │ 1-rc2    │ docker       │ running │
│ foobar       │ 10700 │ 1.0-rc.2 │ remote       │ running │
│ edgedb_azure │ 10767 │ ?        │ remote       │ unknown │
│ zzzzz        │ 10768 │ ?        │ remote       │ unknown │
└──────────────┴───────┴──────────┴──────────────┴─────────┘
```

- EdgeDB CLI Version: EdgeDB CLI 1.0.0-rc.2+d20211123.g786c626b8
- OS Version: ubuntu 21.04

**Comment by fmoor** [comment]
`edgedb instance list --extended` prints the same note about only showing portable packages, but it also includes my docker instances.

**Comment by elprans** [comment]
The explanation text needs clarification.

**Comment by tailhook** [comment]
Should be fixed by #581
